CONTRIBUTING.md

```diff
@@ -10,8 +10,8 @@ Instructions for contributing to [plaid-node][1]. A node.js client library for t
   cp .env.example .env
   ```
 
-2. Go to the [Plaid Dashboard](https://dashboard.plaid.com/) and copy and paste your `client_id`, `public_key`, and `secret`
-   into `.env` using a text editor of your choice. Your account must be enabled for sandbox access.
+2. Go to the [Plaid Dashboard](https://dashboard.plaid.com/) and copy and paste your `client_id`, `public_key`, and sandbox `secret`
+   into `.env` using a text editor of your choice.
```

Comment:
All accounts are enabled for sandbox so this wording didn't make sense to me, and "copy paste your secret" was not specific as all new accounts have multiple secrets.

Reply:
👍